tests/test_task_worker.py

```diff
@@ -151,6 +173,11 @@ async def test_task_worker_handles_aborted_task_run_submission(
 async def test_task_worker_handles_deleted_task_run_submission(
     foo_task, prefect_client, caplog
 ):
+    if PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_ORCHESTRATION:
```

Comment:
I think we could use `pytest.mark.skipif` for these maybe?

Reply:
Cause a fixture is setting `PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_ORCHESTRATION` to be parameterized (once True, once False) I think we might need the test setup to happen?

Is there a good way to reference a fixture in the skipif?